WooCommerce/WooCommerceTests/POS/ViewModels/ItemListViewModelTests.swift

```diff
@@ -24,27 +24,27 @@ final class ItemListViewModelTests: XCTestCase {
         super.tearDown()
     }
 
-    func test_itemListViewModel_when_populatePointOfSaleItems_is_called_then_items_are_populated() async {
+    func test_itemListViewModel_when_loadInitialItems_then_items_are_populated() async {
```

Comment:
super nit, no need to change: since the SUT is the same as the name of the test class (`ItemListViewModel`), I think it's fine to skip `itemListViewModel` in the test case name. It can make the test case easier to read.